src/pages/en/guides/images.md

```diff
@@ -135,35 +126,42 @@ Image files in your project's source directory can be imported in frontmatter an
 
 #### Remote Images
 
-Remote images require a full URL as the image `src`. Also, you must either provide `width` and `height`, or one of the dimensions plus the required `aspectRatio` to the `<Image />` component.
+To use a remote image, pass a full URL to the `<Image />`'s src attribute. `<Image />` won't infer dimensions and format from this remote file. You must provide the `format` to render the image with, and you must either provide `width` and `height` or one of the two dimensions plus an `aspectRatio`.
 
 #### Examples
 
 ```astro
 ---
 // src/pages/index.astro
 import { Image } from '@astrojs/image/components';
-import localImage from '../assets/local.png';
-const imageUrl = 'https://astro.build/assets/logo.png';
+import localImage from "../assets/logo.png";
+const remoteImage = "https://picsum.photos/id/957/300/200.jpg";
+const localAlt = "The Astro Logo";
+const remoteAlt = "A low-angle view of a forest during the daytime";
 ---
 
-// optimized local image, keeping the original width, height, and image format
-<Image src={localImage} />
+<!--optimized local image, keeping the original width, height, and image format-->
+<Image src={localImage} alt={localAlt} />
+
+<!-- height will be recalculated to match the original aspect ratio-->
+<Image src={localImage} width={300} alt={localAlt} />
 
-// height will be recalculated to match the original (local only) or specified aspect ratio
-<Image src={localImage} width={300} />
-<Image src={imageUrl} width={300} aspectRatio={16/9} />
+<!--For remote images, the desired dimensions and format are required-->
+<Image src={remoteImage} width={300} aspectRatio="1:1" format="png" alt={remoteAlt} />
 
-// cropping to a specific width and height
-<Image src={localImage} width={300} height={600} />
-<Image src={imageUrl} width={544} height={184} />
+<!-- cropping to a specific width and height -->
+<Image src={localImage} width={300} height={600} alt={localAlt}/>
+<Image src={remoteImage} width={544} height={184} format="png" alt={remoteAlt}/>
 
-// cropping to a specific aspect ratio and converting to an avif format
-<Image src={localImage} aspectRatio="16:9" format="avif" />
-<Image src={imageUrl} height={200} aspectRatio="16:9" format="avif" />
+<!-- cropping to a specific aspect ratio and converting to an avif format-->
+<Image src={localImage} aspectRatio="16:9" format="avif" alt={localAlt}/>
+<Image src={remoteImage} height={200} aspectRatio="16:9" format="avif" alt={localAlt}/>
 
-// local image imports can also be inlined directly
-<Image src={import('../assets/local.png')} />
+<!-- local image imports can also be inlined directly-->
+<Image src={import('../assets/logo.png')} alt={localAlt}/>
+
+<!-- If an image is stored in the `/public` folder, use its path relative to `/public`-->
+<Image src="penguin.jpg" width="300" aspectRatio={1} format="png" alt="A happy penguin"/>
```

Comment:
```suggestion
<Image src="/penguin.jpg" width="300" aspectRatio={1} format="png" alt="A happy penguin"/>
```
Do we need this?

Reply:
It works either way. Let's add it in to match the code sample in an earlier section

Reply:
Interesting! If it works either way, then I guess we decide how we want to handle it. There's something to be said for NOT using the / if it's not required, to distinguish it from something in `src`.